packages/framework/http/tools/create-http/templates/base/.env.example

```
@@ -3,4 +3,6 @@
 
 NODE_ENV=development
 PORT=3000
+# Comma-separated log levels: error,warn,info,http,verbose,debug,silly
+LOG_LEVELS=error,warn,info
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_LEVELS key should go before the NODE_ENV key

(UnorderedKey)